annotation-processor/src/test/java/org/hibernate/validator/ap/testmodel/annotationparameters/InvalidDigitsParameters.java

```diff
@@ -23,7 +23,27 @@
 	@Digits( integer = -5, fraction = -3 )
 	private BigDecimal decimal3;
 
-	@Digits.List({ @Digits(integer = -5, fraction = -3), @Digits(integer = 5, fraction = -3) })
+	@Digits.List( {@Digits( integer = -5, fraction = -3 ), @Digits( integer = 5, fraction = -3 )} )
 	private BigDecimal decimal4;
 
+	public InvalidDigitsParameters(
+			@Digits( integer = -1, fraction = 3 ) BigDecimal decimal1,
```

Comment:
We're nearly there! We have a space after `{` in the annotations and it looks like the Intellij formatter removes it.

It should be:
```java
@Digits.List({ @Digits(integer = -5, fraction = -3), @Digits(integer = 5, fraction = -3) })
```

If it's an issue with the formatter we provide, could you fix it and open a PR for it?

Once this one is solved, I think we won't have any formatting issue left! :)